cli_reference/developer_cli_odo/installing-odo.adoc

```diff
@@ -6,23 +6,23 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-The following section describes how to install `{odo-title}` on different platforms using the CLI or the Visual Studio Code (VS Code) IDE.
+// The following section describes how to install `{odo-title}` on different platforms using the CLI or the Visual Studio Code (VS Code) IDE.
```

Comment:
IBMG SG pg 22: avoid self referential language, like `This section describes` 

Could you simply say, `You can install {odo-title} on different platforms using the CLI or the Visual Studio Code (VS Code) integrated development environment (IDE)` or `Install {odo-title} on different platforms...`

Note, if this is a standard intro that openshift  docs use, don't change it. You want to be consistent with the other docs' sections. But if you do have the leeway to change it, consider dropping the self referential language.

Reply:
Also, for some reason, I don't see this intro sentence in the preview link:
<img width="1331" alt="Screen Shot 2022-01-19 at 2 36 35 PM" src="https://user-images.githubusercontent.com/70717303/150201518-2e69ae2a-d44e-49ac-bc57-f353e0c3df7e.png">

Reply:
The line including "The following section describes..." is actually commented out in the source for this precise reason - I will delete the comment entirely, for clarity